content/tracing/faq/agent-5-tracing-setup.md

```diff
@@ -0,0 +1,70 @@
+---
+title: APM and Distributed Tracing with agent v5
+kind: faq
+private: true
+---
+
+## Getting started
+
+APM is available as part of the Datadog Agent with versions 5.11+ as part of the one line install for the Linux and Docker Agents. Currently, [Mac](https://github.com/DataDog/datadog-trace-agent#run-on-osx) and [Windows](https://github.com/DataDog/datadog-trace-agent#run-on-windows) users must perform a manual install of the APM Agent (aka Trace Agent) via a separate install process.
+
+The Agent can be enabled by including the following in your [Datadog agent configuration file](/agent/faq/where-is-the-configuration-file-for-the-agent):
+```
+apm_enabled: yes
+```
+
+<div class="alert alert-info">
+APM is enabled by default after Datadog agent 5.13 (on Linux and Docker), and can be disabled by adding the parameter: <code>apm_enabled: no</code> in your Datadog agent configuration file.
+</div>
+
+### Installing the agent
+
+With our infrastructure monitoring, metrics are sent to the Datadog Agent, which then forwards them to Datadog. Similarly, tracing metrics are also sent to the Datadog agent. To enable tracing:
```

Comment:
`also sent to the Datadog agent` => `also sent to the Datadog Agent`

Reply:
👌